### DISCUSSION TITLE
When we use TestHelper class in my unit test class. It is showing as undefined class

### DISCUSSION THREAD

**Discussion by Ravin507** [pimcore/pimcore#11803] [Q&A / Support]
When we try to use TestHelper class into my pimcore application to write unit test using codeception. It is showing as undefined class for TestHelper. Could anyone please suggest

**Comment by dvesh3** [comment]
you have to autoload `Pimcore\Tests` namespace in your `_bootstrap.php`. please follow the documentation here https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Testing/Application_Testing.html#page_Codeception

**Reply by Ravin507** [reply]
@devsh3 I have followed the document and done the necessary changes but still get getting the same issue as below

<img width="584" alt="Screenshot 2022-03-30 at 5 01 56 PM" src="https://user-images.githubusercontent.com/101259879/160825342-9a49ebed-8ecf-4684-80f1-b6a2e30df024.png">
>

**Reply by dvesh3** [reply]
seems like you forgot to import the class using `use Pimcore\Tests\Util\TestHelper;`

**Reply by Ravin507** [reply]
done the import -> use Pimcore\Tests\Util\TestHelper
<img width="1174" alt="Screenshot 2022-03-30 at 5 41 28 PM" src="https://user-images.githubusercontent.com/101259879/160831619-96282cd4-b18d-487d-9289-e64d068c5123.png">

;

**Reply by dvesh3** [reply]
add this in your root composer.json and dump composer autoload:
![image](https://user-images.githubusercontent.com/5137917/160836011-16909921-6444-4bd4-911f-748f9dd2de01.png)

**Reply by Ravin507** [reply]
Added to composer and ran the dump composer autoload, but still getting the same issue @dvesh3

**Reply by Ravin507** [reply]
Hi @dvesh3 While am testing MailTest.php. I am getting the below error. Could you please suggest more on this.
<img width="1147" alt="Screenshot 2022-04-06 at 10 22 49 AM" src="https://user-images.githubusercontent.com/101259879/161898665-aac6c0de-7a38-44c3-b52b-25790b97c79b.png">
